.github/workflows/validate.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Unit Tests
         run: yarn test
       - name: Acceptance Tests
-        run: bash -x aatp/run
+        run: PEERBOOK_REF=subs WEBEXEC_REF=subs bash -x aatp/run
```

Comment:
You need to use workflow parameter with a default of `master` so we can set it when starting the workflow